homepage/templates/homepage/index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+{% load static %}
+<html style="font-size: 16px;">
+
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta charset="utf-8">
+  <meta name="keywords" content="Welcome to, Hotails">
+  <meta name="description" content="">
+  <meta name="page_type" content="np-template-header-footer-from-plugin">
+  <title>Hotails</title>
+  <link rel="stylesheet" href="{% static 'nicepage.css' %}" media="screen">
+  <link rel="stylesheet" href="{% static 'Home.css' %}" media="screen">
+  <meta name="generator" content="Nicepage 4.6.5, nicepage.com">
+  <link id="u-theme-google-font" rel="stylesheet"
+    href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i|Open+Sans:300,300i,400,400i,500,500i,600,600i,700,700i,800,800i">
+  <link id="u-page-google-font" rel="stylesheet"
+    href="https://fonts.googleapis.com/css?family=Roboto+Slab:100,200,300,400,500,600,700,800,900">
+
+  <meta name="theme-color" content="#478ac9">
+  <meta property="og:title" content="Home">
+  <meta property="og:type" content="website">
+</head>
+
+<body data-home-page="index.html" data-home-page-title="Home" class="u-body u-xl-mode">
+  <header class="u-clearfix u-header u-header" id="sec-23af"><a href="" class="u-image u-logo u-image-1"
+      data-image-width="300" data-image-height="300">
+      <img src="{% static 'images/Hotails-logo.jpg' %}" class="u-logo-image u-logo-image-1">
+    </a></header>
+  <section class="u-clearfix u-palette-1-base u-section-1" id="carousel_bf01">
+    <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
+      <div class="u-layout">
+        <div class="u-layout-row">
+          <div
+            class="u-align-center-lg u-align-center-sm u-align-center-xl u-align-center-xs u-black u-container-style u-layout-cell u-left-cell u-size-30 u-layout-cell-1">
+            <div class="u-container-layout u-container-layout-1">
+              <h2 class="u-align-center-md u-text u-text-1">Welcome to</h2>
+              <h1 class="u-align-center-md u-text u-text-2">Hotails</h1>
+              <p class="u-align-center-md u-text u-text-grey-30 u-text-3">An online marketplace that connects people who
+                are looking for a daycare for their dogs,<br>&nbsp; &nbsp; &nbsp; &nbsp; with people want to keep other
+                people dogs for money.<br>
```

Comment:
this sentence doesn't sound good to me (keep dogs for money :( ) 
maybe something like that Hotails main goal is to provide both for dogs owners and dog daycares managers,
an efficient, comfortable, and reliable system.

Reply:
Sounds good
I'll edit it now